benchmarks/benchmarks.py

```diff
@@ -13,12 +13,27 @@
     ModalLocation,
     Flows,
     TotalLocationEvents,
+    subscriber_location_cluster,
+    EventScore,
+    MeaningfulLocations,
+    MeaningfulLocationsAggregate,
+    MeaningfulLocationsOD,
 )
 from .utils import get_env_var
 from .config import FLOWDB_CONFIGS, FLOWDB_CONFIG_PARAM_NAMES
 from .flowdb_config import FlowDBConfig
 
 
+def make_params(params_dict):
+    """
+    Return a list of params and a list of their names, including the FlowDB
+    params defined in config.py and any additional params in params_dict.
+    """
+    params = [list(set(x)) for x in zip(*FLOWDB_CONFIGS)] + list(params_dict.values())
+    param_names = FLOWDB_CONFIG_PARAM_NAMES + list(params_dict.keys())
```

Comment:
Just to check: is it best to pass the FlowDB params first or last? In the benchmarks below we extract the benchmark-specific params as `args[-2]` for example, and the negative number is needed because the relevant params are at the end of the list. Would it be more intuitive to read if we passed them first? Or is this not possible?

Reply:
I'm also not entirely clear where the FlowDB params are actually used in the benchmarks?

Reply:
I agree that the setup() function in each class would be nicer if we didn't need to use negative indices. On the other hand, the order of the params controls the order in which they are run (and displayed), so putting the benchmark-specific params first would mean that consecutive benchmarks would run with the same params on different FlowDB configs, rather than the same FlowDB config with different benchmark-specific params. I don't think it makes any difference at the moment, but I wonder whether this may have an impact in future refactoring.

Reply:
Ok, best to leave it as it is for the moment and address any further tweaks in future PRs.